docs/framework/20categories/20architecture/token-bridges.md

```diff
@@ -1,22 +1,38 @@
-
 ### Token Bridges
-Token bridges are protocols that enable the transfer of assets from one network 
-to another. This can be enacted using either a lock-and-mint or a burn-and-mint 
-mechanism. In both cases, the protocol involves minting synthetic assets on a 
-destination that are claimable for a corresponding amount of underlying assets 
-at a later time. The invariant such protocols must preserve is that the value 
-of an asset only resides in one place at a given time and that the total 
-supply of synthetic assets matches the total underlying. Scenarios in 
-which either locked assets on a source network are stolen or synthetic 
-assets are arbitrarily minted on a destination network, break these 
-invariants. This creates a perpetual risk for token holders of 
-synthetic assets, in which these assets could become worthless 
-in the event of such failures.
-
-Moreover, lock-and-mint token bridges in particular create 
-honey pots that make them an ideal target for malicious 
-actors. A significant portion of bridge hacks over the 
-last couple of years has involved this source of risk.
+One of the most common cross-chain use cases is the [transfer of assets](../../01intro/introduction.md#types-of-crosschain-interaction) from one network to another. This enables an asset in one network to be used inside applications in another. For instance, a user might want to transfer her ETH from Ethereum to Avalanche so she can use it as collateral in a lending protocol on Avalanche. A token bridge is a protocol that enables this requirement. It often relies on an underlying messaging protocol for cross-chain message communication. 
```

Comment:
crosschain

Reply:
This change from cross-chain to crosschain is needed in multiple places.

Reply:
habit :) .. fixed